app/pages/sections/order-items/catalogue-solutions/routes.test.js

```diff
@@ -164,138 +161,143 @@ describe('catalogue-solutions section routes', () => {
+  //         .send({ _csrf: csrfToken })
+  //         .expect(200)
+  //         .then((res) => {
+  //           expect(res.text.includes('data-test-id="order-item-page"')).toEqual(true);
+  //           expect(res.text.includes('data-test-id="error-summary"')).toEqual(true);
+  //           expect(res.text.includes('data-test-id="error-title"')).toEqual(false);
+  //         });
+  //     });
+
+  //     it('should show the catalogue-solutions
+  // order item page with errors if the api response is unsuccessful', async () => {
+  //       getOrderItemPageData.mockResolvedValue({});
+  //       orderItemController.getOrderItemContext = jest.fn().mockResolvedValue({});
+  //       validateOrderItemForm.mockReturnValue([]);
+  //       saveOrderItem.mockResolvedValue({ success: false, errors: [{}] });
+  //       orderItemController.getOrderItemErrorPageContext = jest.fn()
+  //         .mockResolvedValue({
+  //           errors: [{ text: 'Select a price', href: '#priceRequired' }],
+  //         });
+
+  //       const { cookies, csrfToken } = await getCsrfTokenFromGet({
+  //         app: request(setUpFakeApp()),
+  //         getPath: path,
+  //         getPathCookies: [mockAuthorisedCookie],
+  //       });
+
+  //       return request(setUpFakeApp())
+  //         .post(path)
+  //         .type('form')
+  //         .set('Cookie', [cookies, mockAuthorisedCookie, mockGetPageDataCookie])
+  //         .send({ _csrf: csrfToken })
+  //         .expect(200)
+  //         .then((res) => {
+  //           expect(res.text.includes('data-test-id="order-item-page"')).toEqual(true);
+  //           expect(res.text.includes('data-test-id="error-summary"')).toEqual(true);
+  //           expect(res.text.includes('data-test-id="error-title"')).toEqual(false);
+  //         });
+  //     });
+
+  //     it('should redirect to /organisation/some-order-id/catalogue-solutions
+  // if there are no validation errors and post is successful', async () => {
+  //       getOrderItemPageData.mockResolvedValue({});
+  //       orderItemController.getOrderItemContext = jest.fn().mockResolvedValue({});
+  //       validateOrderItemForm.mockReturnValue([]);
+  //       saveOrderItem.mockResolvedValue({ success: true });
+
+//       const { cookies, csrfToken } = await getCsrfTokenFromGet({
+//         app: request(setUpFakeApp()),
+//         getPath: path,
+//         getPathCookies: [
+//           mockAuthorisedCookie,
+//         ],
+//       });
+//       return request(setUpFakeApp())
+//         .post(path)
+//         .type('form')
+//         .set('Cookie', [cookies, mockAuthorisedCookie, mockGetPageDataCookie])
+//         .send({
+//           _csrf: csrfToken,
+//         })
+//         .expect(302)
+//         .then((res) => {
+//           expect(res.redirect).toEqual(true);
+//           expect(res.headers.location).toEqual
+// (`${baseUrl}/organisation/some-order-id/catalogue-solutions`);
+//         });
+//     });
+//   });
 });
```

Comment:
What's your intention with the commented out code?

Reply:
it's going to be needed for the next story so didn't want to delete it only to re add it